force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -202,6 +230,13 @@
         <shortDescription>Error capturing metrics.</shortDescription>
         <value>We couldn't collect telemetry metrics.</value>
     </labels>
+    <labels>
+        <fullName>Total_Records</fullName>
```

Comment:
I sent a message this morning to Ashwin, I would like us to use the same title on all of the data tables for consistency and also so we do not need to introduce extra labels.

Reply:
@jjbennett, If you are referring to this label then it is used to show the total number of sessions that will be created. Let me know what Ashwin says and we can make the necessary changes to this one.

Reply:
After speaking with Ashwin, we will keep Total but we will use the SObject Plural label vs "Sessions" alone.